src/components/Dashboard/index.jsx

```diff
@@ -87,6 +88,9 @@ import ErrorPanel from '../ErrorPanel';
       paddingRight: theme.spacing.triple,
       paddingBottom: theme.spacing.triple * 4,
     },
+    logoStyle: {
```

Comment:
You'll have to add `width: '100%',` to `title`, otherwise the hover won't work since the image doesn't take the full width.

```diff
title: {
  textDecoration: 'none',
  color: theme.palette.text.primary,
+ width: '100%',
},
```

Reply:
Ahh yes I encountered this problem too and didn't know how to solve it thanks for the help!

Reply:
No worries!

Reply:
@helfi92 hopefully it works 😅